src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -475,9 +475,17 @@ private static FieldFrame getObjectFrame(FieldFrame frame) {
      */
     private static boolean checkLineNo(DetailAST field, DetailAST objCalledOn) {
         boolean result = false;
+        // We should specify columnNo passing condition in such a way,
+        // so that the minimal possible distance between field and objCalledOn
+        // will be the maximal condition to pass this check.
+        // The minimal distance between objCalledOn and field (of type String) initialisation
+        // is calculated as follows:
+        // String(6) + space(1) + variableName(1) + assign(1) +
+        // anotherStringVatiableName(1) + semicolumn(1) = 11
```

Comment:
anotherStringVatiableName is still mispelled

I do not see exmaple line that I requested to be added.

Reply:
done, mixed up checks